plugins/view-transitions/js/types.ts

```diff
@@ -1,8 +1,14 @@
-export type ViewTransitionsConfig = {
+export interface ViewTransitionAnimationConfig {
+	useGlobalTransitionNames: boolean;
+	usePostTransitionNames: boolean;
+}
+
+export interface ViewTransitionsConfig {
```

Comment:
Why `interface` instead of `type`?

Reply:
FYI I fixed this in https://github.com/WordPress/performance/pull/2025/commits/95d0b4da6a3cffd59b40402f567b3a15d1f74029 because it's a bug specific to that plugin. For now I used `type` for consistency, we can reconsider later / as part of this PR.

Reply:
I've seen various takes on that, don't have an opinion really, just not sure what's best. What I recollect reading about this was that it doesn't really matter and "just choose one and stick with it".